c2cgeoportal/views/export.py

```diff
@@ -40,7 +40,7 @@
 
 
 @view_config(route_name="csvecho")
-def echo(request):
+def exportcsv(request):
```

Comment:
I don't like to use different name for function and route.
Can we put everywhere `exportcsv`?